cmd/dlt/accountroles/cmd.go

```diff
@@ -158,17 +158,8 @@ func run(cmd *cobra.Command, _ []string) {
 
 	if r.Creator.IsGovcloud {
```

Comment:
@Mischulee I think we can remove the `else if` part of this statement. The `setDeleteRoles` function will return `true` for the assignment to `deletedHostedCP` if either:

- both `--classic` and `--hosted-cp` are omitted
- `--hosted-cp` is explicitly set to true

Otherwise, it should remain `false`.

Do you agree?

Reply:
@robpblake Yes, this probably makes the most sense. Fixed.